config/environments/production.rb

```diff
@@ -83,18 +83,19 @@
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
   # Mailer settings
-  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.delivery_method = :ses
   config.action_mailer.default_url_options = { host: "https://circuitverse.org/" }
 
-  config.action_mailer.smtp_settings = {
-    :user_name => 'apikey',
-    :password => ENV["CIRCUITVERSE_EMAIL_PASSWORD"],
-    :domain => 'circuitverse.org',
-    :address => 'smtp.sendgrid.net',
-    :port => 587,
-    :authentication => :plain,
-    :enable_starttls_auto => true
-  }
+  # config.action_mailer.delivery_method = :smtp
+  # config.action_mailer.smtp_settings = {
+  #   :user_name => 'apikey',
+  #   :password => ENV["CIRCUITVERSE_EMAIL_PASSWORD"],
+  #   :domain => 'circuitverse.org',
+  #   :address => 'smtp.sendgrid.net',
+  #   :port => 587,
+  #   :authentication => :plain,
+  #   :enable_starttls_auto => true
+  # }
```

Comment:
Commented out send-grid creds for now. Should I remove it?

Reply:
why not